### ISSUE TITLE
openMV

### ISSUE THREAD

**Issue by halilkalayli** [sipeed/MaixPy-v1#259]
maixpy’s openmv implementation is not up-to-date. there are functions that is not implemented. when will they updated?

**Comment by easy-and-simple** [comment]
No such thing like openmv
There are openmv ide and openmv cam and some openmv api that is specific to openmv cam boards.
Maybe you will tell us what you call openmv and what functions are not up to date and how you found that if maixpy documentation is so bad and incomplete?

**Comment by halilkalayli** [comment]
Sorry my mistake. I wanted to say openMV library. 
for example:
- image.crop() function returns AttributeError: 'Image' object has no attribute 'crop'
- blob.corners() function returns AttributeError: 'blob' object has no attribute 'corners'
- blob.min_corners() function returns AttributeError: 'blob' object has no attribute 'min_corners'
- image.rotation_corr() has no option for "fov" and "corners"

my setup:
- sipeed m1 dock suit
- MaixPy IDE 0.2.5
- firmware: maixpy_v0.5.0_88

**Comment by easy-and-simple** [comment]
You are right that there are a lot of things that should be fixed, implemented and completed, but I think your problem is in bad documentation.
I got similar errors when tried to use functions directly on classes, but it is working when you assign object to variable and then use desired function. 
I don't know if there is crop function for image because in examples I have seen cut function, but if there is function crop you should not call it directly as in your example - image.crop, but should use something like var1=image.image and then var1.crop. I hop you got the idea

**Comment by halilkalayli** [comment]
Of course there are some documentation problems. But what i wanted to point is openMV has updates (as seen from attachments) and sipeed did not implement them. 

![Screenshot from 2020-06-01 01-03-29](https://user-images.githubusercontent.com/62602439/83363759-8801ba00-a3a4-11ea-8c19-6a497434beb6.png)
![Screenshot from 2020-06-01 01-03-34](https://user-images.githubusercontent.com/62602439/83363760-8932e700-a3a4-11ea-9a7d-991abfde71db.png)

**Comment by junhuanchen** [comment]
Mark, these interfaces are really not implemented.

**Comment by FPSychotic** [comment]
Please add the thermal camera drivers that openmv support
